src/movingaverages.jl

```diff
@@ -76,6 +76,26 @@ function kama(ta::TimeArray{T,N}, n::Int=10, fn::Int=2, sn::Int=30) where {T,N}
     TimeArray(cl.timestamp, vals, cols)
 end
 
+function env(ta::TimeArray{T,N}, n::Int; e::Float64 = 0.1) where {T,N}
+    tstamps = ta.timestamp[n:end]
+
+    s = sma(ta, n)
+
+    upper = s.values .+ (s.values .* e)
+    lower = s.values .- (s.values .* e)
```

Comment:
I think the broadcast of TimeArray works here.
```julia
upper = @. s + (s * e)
lower = @. s - (s * e)
```

Reply:
It works with `s.values`.